src/Api/Controller/AbstractSerializeController.php

```diff
@@ -197,6 +225,106 @@ protected function buildParameters(ServerRequestInterface $request)
         return new Parameters($request->getQueryParams());
     }
 
+    /**
+     * Set the serializer that will serialize data for the endpoint.
+     *
+     * @param string $serializer
+     */
+    public function setSerializer(string $serializer)
```

Comment:
Maybe we should add @internal to all these?

Reply:
I'm thinking there might be cases where these methods are directly used inside a callback passed to the extender's `prepareDataQuery`, not sure though

Reply:
Hmm fair point, someone might want to add variable imports.